app/jobs/leader_check_ins_job.rb

```diff
@@ -28,7 +28,7 @@ def perform(streak_keys = [])
 
   private
 
-  def retrieive_user_ids(streak_keys)
+  def retrieve_user_ids(streak_keys)
```

Comment:
@paked This will already be covered in https://github.com/hackclub/api/pull/116/files#diff-901c18c30d3bc30f13b107f181a0f6fdL31.